plugins/nginx_logs.yaml

```diff
@@ -0,0 +1,129 @@
+version: 0.0.1
+title: NGINX
+description: Log parser for NGINX
+parameters:
+  # The observiq log format is defined as:
+  # log_format observiq '{"remote_addr":"$remote_addr","remote_user":"$remote_user","time_local":"$time_local","request":"$request","status":"$status","body_bytes_sent":"$body_bytes_sent","http_referer":"$http_referer","http_user_agent":"$http_user_agent","request_length":"$request_length","request_time":"$request_time","upstream_addr":"$upstream_addr","upstream_response_length":"$upstream_response_length","upstream_response_time":"$upstream_response_time","upstream_status":"$upstream_status","proxy_add_x_forwarded_for":"$proxy_add_x_forwarded_for","bytes_sent":"$bytes_sent","time_iso8601":"$time_iso8601","upstream_connect_time":"$upstream_connect_time","upstream_header_time":"$upstream_header_time","http_x_forwarded_for":"$http_x_forwarded_for"}';
+  # See https://docs.nginx.com/nginx/admin-guide/monitoring/logging/#setting-up-the-access-log for more information on configuring log format
+  - name: log_format
+    type: string
+    supported:
+      - default
+      - observiq
+    default: default
+  - name: enable_access_log
+    type: bool
+    default: true
+  - name: access_log_paths
+    type: "[]string"
+    default: 
+      - "/var/log/nginx/access.log*"
+  - name: enable_error_log
+    type: bool
+    default: true
+  - name: error_log_paths
+    type: string
+    default: 
+      - "/var/log/nginx/error.log*"
+  - name: start_at
+    type: string
+    supported:
+      - beginning
+      - end
+    default: end
+  - name: encoding
+    type: enum
+    supported:
+      - nop
+      - utf-8
+      - utf-16le
+      - utf-16be
+      - ascii
+      - big5
+    default: utf-8
+template: |
+    receivers:
+      {{ if .enable_access_log }}
+      filelog/access_log:
+        include:
+          {{ range $fp := .access_log_paths }}
+          - '{{ $fp }}'
+          {{end}}
+        attributes:
+          log_type: 'nginx.access'
+        start_at: {{ .start_at }}
+        encoding: {{ .encoding }}
+        operators:
+          {{ if eq .log_format "default" }}
+          - type: regex_parser
+            regex: '^(?P<http_request_remoteIp>[^ ]*) (?P<host>[^ ]*) (?P<user>[^ ]*) \[(?P<time_local>[^\]]*)\] "(?P<http_request_requestMethod>\S+)(?: +(?P<http_request_requestUrl>[^\"]*?)(?: +(?P<http_request_protocol>\S+))?)?" (?P<http_request_status>[^ ]*) (?P<http_request_responseSize>[^ ]*)(?: "(?P<http_request_referer>[^\"]*)" "(?P<http_request_userAgent>[^\"]*)")?$'
+            timestamp:
+              parse_from: "$body.time_local"
```

Comment:
I hit this issue with other plugins but should this be `$body.time_local`? I had to use `attributes.time_local` when the collector was using the latest logging library.

Reply:
Same question everywhere `$body` exists.

Reply:
I think you're right, I was testing with v0.5.0, not latest.

Reply:
Fixed this and tested with the latest collector; Thanks for pointing this out, didn't realise those breaking changes weren't yet made on our latest release.